### DISCUSSION TITLE
button push functionality

### DISCUSSION THREAD

**Discussion by phongshader** [IBM/mac-ibm-notifications#222] [Q&A] [answered]
Is this possible to have "System Preferences" button run `open "x-apple.systempreferences:com.apple.preference.security?Privacy_ScreenCapture"`
<img width="531" alt="Screenshot 2024-09-05 at 1 30 40 PM" src="https://github.com/user-attachments/assets/69136d91-df8d-4502-a96c-0328aa0b9a22">

**Comment by SMartorelli** [comment, marked as answer]
Sure, just define the main button CTA with `-main_button_cta_type link -main_button_cta_payload "x-apple.systempreferences:com.apple.preference.security?Privacy_ScreenCapture"`
